### PR TITLE
Replace football with world cup in main nav

### PR DESCRIPTION
**Pull request by nicl**
This applies across editions.

~~It lives behind a switch and should be enabled before the world cup and then disabled afterwards.~~

Note, because navigation is not recreated on each request, a deploy is required to update the switch value unfortunately.

**UPDATE: editorial have decided to put world cup *and* football on the main news subnav. And also to promote football on some other subnavs to live next to world cup.**

### CODE REVIEW

**Review by NataliaLKB — APPROVED** [review]

**Comment by nicl** [comment]
@NataliaLKB hey this has changed as editorial have changed their minds. Are you happy to re-review? Thanks!

File: common/app/navigation/Navigation.scala [hunk 1]
```diff
@@ -159,20 +159,40 @@ object NavMenu {
   }
 
   private[navigation] def getSubnav(page: Page, currentNavLink: Option[NavLink], currentParent: Option[NavLink], currentPillar: Option[NavLink]): Subnav = {
+    //log.info(s"YO! $page \n\n $currentNavLink \n\n $currentParent \n\n $currentPillar")
```
Comment (NataliaLKB) [inline common/app/navigation/Navigation.scala]: Does all this need to be there?

Comment (nicl) [reply common/app/navigation/Navigation.scala]: erm

Comment (nicl) [reply common/app/navigation/Navigation.scala]: (None of it does - will remove.)

**Review by NataliaLKB — APPROVED** [review]
🌟

**Pull request merged**